README.md

```diff
@@ -27,6 +27,7 @@ When using this together with `esp-println` make sure to use the same output kin
 | exception-handler | Include an exception handler, will add `esp-println` as a dependency |
 | print-uart        | Use UART to print messages\*                                         |
 | print-jtag-serial | Use JTAG-Serial to print messages\*                                  |
+| defmt-espflash    | Use `defmt` logging to print messages\* (check [example](https://github.com/playfulFence/backtrace-defmt-example))                    |
```

Comment:
Please, update the `esp-backtrace` to point at crates.io now that its released, or at least to github